<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NesEmulateWindow</class>
 <widget class="QMainWindow" name="NesEmulateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NesEmulate Author nejidev</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="NesScreenWidget" name="nesScreenWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSeting">
    <property name="title">
     <string>Seting</string>
    </property>
    <addaction name="actionTrueColour"/>
    <addaction name="actionGray"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSeting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTrueColour">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>True colour</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NesScreenWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>NesScreenWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
